pkg/apis/admissionregistration/types.go

```diff
@@ -566,6 +566,19 @@ type ValidatingWebhook struct {
 	// does not understand, calls to the webhook will fail and be subject to the failure policy.
 	// +optional
 	AdmissionReviewVersions []string
+
+	// MatchConditions is a list of conditions on the AdmissionRequest ('request') that must be met
+	// for a request to be sent to this webhook. All conditions in the list must evaluate to TRUE for
+	// the request to be matched. This is an alpha feature and managed by the AdmissionWebhookMatchConditions feature gate.
```

Comment:
Instead of  `This is an alpha feature and managed by the AdmissionWebhookMatchConditions feature gate`, I think you can add a `+featureGate=AdmissionWebhookMatchConditions` at the bottom of this comment block instead. See example: https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/api/core/v1/types.go#L545

Reply:
done